source/fundamentals/builders/sort.txt

```diff
@@ -261,82 +195,13 @@ in the following order:
-Use the ``Sorts.metaTextScore()`` static factory method to build your sort
-criteria to sort by the text score.
-
-.. warning:: Make Sure to Create a Text Index
-
-   You need a :manual:`text index </core/index-text/>` on your collection to perform a text search. See the server manual documentation for more
-   information on how to
-   :manual:`create a text index </core/index-text/#create-text-index>`.
-
-In the following code example, we show how you can use the
-``Sorts.metaTextScore()`` method to sort the results of a text
-search on the :ref:`sort_example collection <sorts-builders-sort-example>`.
-The code example uses the :doc:`Filters </fundamentals/builders/filters>`,
-:doc:`Indexes </fundamentals/builders/indexes>`, and
-:doc:`Projections </fundamentals/builders/projections>` builders.
-The code example performs the following actions:
-
-#. Creates a text index for your
-   :ref:`sort_example collection <sorts-builders-sort-example>`
-   on the ``food`` field.
-#. Runs your text search for the phrase "maple donut".
-#. Projects text scores into your query results as the
-   ``score`` field. This projection is optional if your MongoDB instance is
-   running MongoDB 4.4 or later. 
-#. Sorts your results by text score (best match first).
-
-.. code-block:: java
-
-   import com.mongodb.client.model.Sorts;
-   import com.mongodb.client.model.Projections;
-   import com.mongodb.client.model.Filters;
-   import com.mongodb.client.model.Indexes;
-
-   // <MongoCollection setup code here>
-   
-   collection.createIndex(Indexes.text("food"));
-   Bson metaTextScoreSort = Sorts.metaTextScore("score");
-   Bson metaTextScoreProj = Projections.metaTextScore("score");
-   String searchTerm = "maple donut";
-   Bson searchQuery = Filters.text(searchTerm);
-   collection.find(searchQuery)
-            .projection(metaTextScoreProj)
-            .sort(metaTextScoreSort)
-            .into(results);
-   for (Document result : results) {
-         System.out.println(result.toJson());
-   }
-
-The output of the code example above should look something like this: 
-
-.. code-block:: json
-
-   {"_id": 6, "letter": "c", "food": "maple donut", "score": 1.5}
-   {"_id": 2, "letter": "a", "food": "donuts and coffee", "score": 0.75}
-   {"_id": 3, "letter": "a", "food": "maple syrup", "score": 0.75}
-
-.. note:: MongoDB 4.4 or later ``$meta`` Behavior
+Text Score
+----------
 
-  When using MongoDB 4.4 or later, projecting ``Projections.metaTextScore()`` 
-  into your ``FindIterable`` instance is not necessary to sort on the text
-  score. In addition, MongoDB 4.4 or later disregards the field name you specify
-  in a ``$meta`` text score aggregation operation used in a sort. This means
-  that the field name argument you pass to ``Sorts.metaTextScore()`` is ignored
-  in MongoDB 4.4 or later.
+To specify a sort by the text score of a text search, use the
+``Sorts.metaTextScore()`` static factory method. For a detailed usage example
```

Comment:
Thanks for the note. I changed this to just "for a detailed example". Going forward, I will try to avoid overloading terms in the documentation. Fix: https://github.com/mongodb/docs-java/pull/66/commits/4ca761f59685fc5ab3e5adc47d36cd702f3548bf